### ISSUE TITLE
Problem: AM17 docs use name Redhat instead of `Red Hat`

### ISSUE THREAD

**Issue by sevein** [artefactual/archivematica-docs#109]
```
$ git grep -l "Redhat"
admin-manual/installation-setup/installation/install-centos.rst
admin-manual/installation-setup/installation/installation.rst
admin-manual/installation-setup/upgrading/upgrading.rst
```

**Comment by ablwr** [comment]
This will be resolved by @sallain in #108.

**Comment by sallain** [comment]
Resolved in #108
